CHANGELOG.asciidoc

```diff
@@ -3,6 +3,121 @@
 :issue: https://github.com/elastic/beats/issues/
 :pull: https://github.com/elastic/beats/pull/
 
+[[release-notes-8.7.0]]
+=== Beats version 8.7.0
+https://github.com/elastic/beats/compare/v8.6.2\...v8.7.0[View commits]
+
+==== Breaking changes
+
+*Filebeat*
+
+- Metrics hosted by the HTTP monitoring endpoint for the `aws-cloudwatch`, `aws-s3`, `cel`, and `lumberjack` inputs are now available under `/inputs/` instead of `/dataset`.
+
+*Winlogbeat*
+
+- Added processing for Windows Event ID's 4797, 5379, 5380, 5381, and 5382 for the Security Ingest Pipeline {issue}34293[34293] {pull}34294[34294]
+- Added processing for Windows Event ID's 5140 and 5145 for the Security Ingest Pipeline {pull}34352[34352]
+
+==== Bugfixes
+
+*Affecting all Beats*
+- Fix namespacing on self-monitoring {pull}32336[32336]
```

Comment:
This is an old changelog entry from a previous release. I think the same is true for the two below as well:

```
- Fix race condition when stopping runners {pull}32433[32433]
- Fix concurrent map writes when system/process code called from reporter code {pull}32491[32491[
```

I recognize these they keep coming back from merge conflicts. If you open the linked PRs they should be quite old.